packages/esm-patient-vitals-app/src/vitals-biometrics-form/vitals-biometrics-form.workspace.tsx

```diff
@@ -93,6 +105,26 @@ const VitalsAndBiometricsForm: React.FC<DefaultPatientWorkspaceProps> = ({
   const [showErrorNotification, setShowErrorNotification] = useState(false);
   const [showErrorMessage, setShowErrorMessage] = useState(false);
 
+  const vitalsBiometricsFormData = vitalsBiometrics?.filter(
+    (vitalsBiometrics) => vitalsBiometrics.uuid === encounterUuid,
+  )[0];
+
+  const defaultValues =
+    formContext === 'editing' && vitalsBiometricsFormData
+      ? {
```

Comment:
I don't really understand why we're creating this object here? Would be harmful to do:

```ts
const defaultValues = formContext === 'editing' ? vitalsBiometricsFormData ?? {} : {};
```

Reply:
The more concise approach you suggested doesn't work as well if `vitalsBiometricsFormData` has nested undefined values. The detailed object construction approach appears to work better when `vitalsBiometricsFormData` is partially populated as it excludes undefined values.